src/inhibitors/requiredSettings.js

```diff
@@ -3,10 +3,17 @@ const { Inhibitor } = require('klasa');
 module.exports = class extends Inhibitor {
 
 	async run(message, command) {
-		if (!command.requiredSettings.length || message.channel.type !== 'text') return;
-		// eslint-disable-next-line eqeqeq
-		const requiredSettings = command.requiredSettings.filter(setting => message.guild.settings.get(setting) == null);
-		if (requiredSettings.length) throw message.language.get('INHIBITOR_REQUIRED_SETTINGS', requiredSettings);
+		const missingSettings = [];
```

Comment:
This is the wrong structure for storing settings from multiple locations. We should not be using tuples here. Object<gateway, Array\<settingName>>